assets/themes/hailhydrant/hailhydrant.json

```diff
@@ -0,0 +1,601 @@
+        },
+        {
+          "question": {
+            "en": "Describe the position of the hydrant."
+          },
+          "render": {
+            "en": "Position: {fire_hydrant:position}"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "fire_hydrant:position=sidewalk"
+                ]
+              },
+              "then": {
+                "en": "Sidewalk"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "fire_hydrant:position=lane"
+                ]
+              },
+              "then": {
+                "en": "Road-side"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "fire_hydrant:position=underground"
+                ]
+              },
+              "then": {
+                "en": "Underground"
+              }
+            }
+          ],
+          "freeform": {
+            "key": "fire_hydrant:position"
+          }
+        },
+        {
+          "render": {
+            "en": "Mapillary: {mapillary}"
+          },
+          "question": {
+            "en": "Do you know the Mapillary photo id?"
+          },
+          "freeform": {
+            "key": "mapillary"
+          }
+        }
+      ],
+      "hideUnderlayingFeaturesMinPercentage": 0,
+      "icon": {
+        "render": "https://upload.wikimedia.org/wikipedia/commons/6/66/Hydrant_02.svg"
+      },
+      "width": {
+        "render": "8"
+      },
+      "iconSize": {
+        "render": "15,15,center"
+      },
+      "color": {
+        "render": "#00f"
+      },
+      "presets": [
+        {
+          "tags": [
+            "emergency=fire_hydrant"
+          ],
+          "title": {
+            "en": "fire hydrant"
+          },
+          "description": {
+            "en": "Add a new fire hydant to the map."
+          }
+        }
+      ],
+      "wayHandling": 1,
+      "osmSource": {
+        "overpassTags": {
+          "and": [
+            "emergency=fire_hydrant"
+          ]
+        }
+      }
+    },
+    {
+      "id": "extinguisher",
+      "name": {
+        "en": "Map of fire extinguishers."
+      },
+      "minzoom": 14,
+      "overpassTags": {
+        "and": [
+          "emergency=fire_extinguisher"
+        ]
+      },
+      "title": {
+        "render": {
+          "en": "Extinguishers"
+        }
+      },
+      "description": {
+        "en": "Map layer to show fire hydrants."
+      },
+      "tagRenderings": [
+        {
+          "render": {
+            "en": "Location: {location}"
+          },
+          "question": {
+            "en": "Where is it positioned?"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "location=indoor"
+                ]
+              },
+              "then": {
+                "en": "Indoor"
```

Comment:
Maybe add `Located indoor` instead of simply `indoor`?

Reply:
Fixed wording